SimCalorimetry/EcalEBTrigPrimProducers/plugins/EcalEBTrigPrimPhase2ESProducer.h

```diff
@@ -0,0 +1,81 @@
+#ifndef SimCalorimetry_EcalEBTrigPrimProducers_EcalEBTrigPrimPhase2ESProducer_H
```

Comment:
This class definition should be in the .cc file and not in a separate header.

Reply:
Why ?

Reply:
CMS code rule 6.7: Do not split plugins into header and source files.
https://cms-sw.github.io/cms_coding_rules.html